### DISCUSSION TITLE
Increasing GPU memory on PI with HASSIO

### DISCUSSION THREAD

**Discussion by muzzak123** [blakeblackshear/frigate#1130] [Ask A Question] [answered]
Hi,
I've got a Raspberry PI 4B with 8 GB and a Coral USB. My camera are RSTP feeds and work fine through VLC.
In Frigate three 1080p cameras work fine but the CPU is at 90% without hardware acceleration. 
When I apply hardware acceleration:
`      hwaccel_args:
        - -c:v
        - h264_v4l2m2m`
two of my cameras go completely green.
I assume I need to increase my GPU memory but how do I do that under HASSIO ?

**Comment by federicotravaini** [comment]
raspi-config

**Reply by v1027m** [reply]
Another question, how do you access raspi-config? I tried the ssh add on in HassOS but it didn't work.
Would I have to put an SD in with Raspberry Pi OS and set it there?

**Reply by federicotravaini** [reply]
Best solution is to run hass in docker so you can use raspi-config

**Reply by muzzak123** [reply]
@federicotravaini Not sure it's a real solution to rebuild your entire system with a new operating system. Like "fix your problem with Windows by installing Linux" !!

**Reply by blakeblackshear** [reply]
You can modify the config.txt on the SD card directly.

**Reply by muzzak123** [reply]
Thank you @blakeblackshear , I never realised that you could do that. All good now.

**Comment by Kate222** [comment]
I use rpi4b 8G with hass and have had the same green screen problem when using either hwaccel_args: - -c:v - h264_v4l2m2m or - h264_mmal. The log message is just saying broken frame. Frigate works fine without hwaccel args. I do not use coral usb. Most of the time my cpu at 30% and temp at 41C for 3 cameras.

**Comment by muzzak123** [comment, marked as answer]
@blakeblackshear answer fixed it

> You can modify the config.txt on the SD card directly.

**Reply by ImLovinIt** [reply]
I wished I checked this thread before I spent ages reading through dev info and configuring SSH to the root folder with public/private authentication keys...
